tests/stm32_bootloader/Makefile

```diff
@@ -0,0 +1,10 @@
+BOARD ?= nucleo-f411re
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = cpu_stm32
```

Comment:
You don't need this line if you use `FEATURES_REQUIRED += bootloader_stm32` instead.

Reply:
Changed it.